packages/contracts/contracts/L1/rollup/StateCommitmentChain.sol

```diff
@@ -49,6 +49,19 @@ contract StateCommitmentChain is IStateCommitmentChain, Lib_AddressResolver, Cro
         SEQUENCER_PUBLISH_WINDOW = _sequencerPublishWindow;
     }
 
+    /**
+ * Modifier to enforce that, if configured, only the Owner may
+ * successfully call a method.
+ */
+    modifier onlyOwner() {
+        require(msg.sender == libAddressManager.owner(), "Only callable by the Owner.");
+        _;
+    }
+
+    function setSequencerPublisWindow(uint256 _sequencerPublishWindow)  external onlyOwner {
```

Comment:
can we require the msg.sender must equal to the BondManager which resolved online? like below, then we don't need to have another owner character
require(
            IBondManager(resolve("BondManager")).isCollateralized(msg.sender),
            "Proposer does not have enough collateral posted"
        );

Reply:
good ideas. Changed.